src/form-validator/validator.ts

```diff
@@ -2,152 +2,151 @@ import { classNames } from "../config/class-name";
 import { config } from "../config/config-attributes";
 
 export class FormValidator {
-
-    isEmailValid(email: string): boolean {
-        const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
-        return emailRegex.test(email);
+  isEmailValid(email: string): boolean {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    return emailRegex.test(email);
+  }
+
+  validationNumber(phoneNumber: string): boolean {
+    const phoneRegex = /^\+\d{8,}$/;
+    return phoneRegex.test(phoneNumber);
+  }
+
+  arePasswordsMatching(): any {
+    const passwordInput = document.querySelector('input[name="Password"]') as HTMLInputElement;
+    const confirmPasswordInput = document.querySelector('input[name="Confirm Password"]') as HTMLInputElement;
+    const password = passwordInput ? passwordInput.value : "";
+    const confirmPassword = confirmPasswordInput ? confirmPasswordInput.value : "";
+    return password === confirmPassword;
+  }
+
+  isStrongPassword(password: string): any {
+    const requirements = [
+      { regex: /.{8,}/, message: config.error.characters },
+      { regex: /[A-Z]/, message: config.error.uppercase },
+      { regex: /[a-z]/, message: config.error.lowercase },
+      { regex: /\d/, message: config.error.digit },
+      { regex: /[!@#$%^&*(),.?":{}|<>]/, message: config.error.character },
+    ];
+    const missingSigns = requirements.filter(({ regex }) => !regex.test(password)).map(({ message }) => message);
+    return missingSigns.join(", ");
+  }
+
+  createErrorParagraph(validationError: string, inputElement: any): HTMLParagraphElement {
+    const errorParagraphId = `${inputElement.name}-error`;
+    let errorParagraph = document.getElementById(errorParagraphId) as HTMLParagraphElement;
+    errorParagraph = document.createElement("p");
+    errorParagraph.id = errorParagraphId;
+    errorParagraph.textContent = validationError;
+    const inputElementNode = inputElement.parentNode as Node;
+    inputElementNode.insertBefore(errorParagraph, inputElement.nextSibling);
+    return errorParagraph;
+  }
+
+  removeErrorParagraph(errorParagraph: HTMLElement): void {
+    if (errorParagraph !== null && errorParagraph !== undefined) {
+      return errorParagraph.remove();
+    } else {
```

Comment:
znowu. Nie potrzebujesz tu else. Sprawdz wszystkie if ktore masz i jezeli sytuacja sie powtarza to popraw wszedzie